xdrip-get-entries.sh

```diff
@@ -135,6 +136,9 @@ main()
 
   if [ "$state" != "Stopped" ] || [ "$mode" != "not-expired" ]; then
     log "Posting glucose record to xdripAPS / OpenAPS"
+    if [ -e "${LDIR}/entry-backfill2.json" ] ; then
+      /usr/local/bin/cgm-post-xdrip ${LDIR}/entry-backfill2.json
```

Comment:
All looks great. So this backfill happens regardless of mode ... expired or not expired. Is that the intention? We may need some checks to make sure that expired is working in dual mode meaning that all the calibrations have been going to the tx as well as local LSR. That is what is supposed to happen. Perhaps we can check that the "$state != "Stopped" in this inner if statement? Open to ideas. I might just need to try it out in expired mode and see.

Reply:
You're right, I didn't consider expired mode at all. I'm not even sure what happens when there is no session.